python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -232,6 +231,40 @@ def __new__(
             has_uninstalled_deps=has_uninstalled_deps,
             manifest_preparer=manifest_preparer,
         )
-        if manifest_preparer:
-            manifest_preparer.on_load(val)
-        return val
+
+    @public
+    def prepare_if_dev(self) -> None:
+        """Prepare a dbt project at run time during development.
+
+        The preparation process ensures that the dbt manifest file and dbt dependencies are available and up-to-date:
+            * During development, pull the dependencies and reload the manifest at run time to pick up any changes.
+            * When deploying, expect a manifest that was created at build time to reduce start-up time.
```

Comment:
Is it relevant to talk about deploying here, given that the name is `if_dev`?

Reply:
Oh no it's not. Now removed in [ff4bdf0](https://github.com/dagster-io/dagster/pull/22513/commits/ff4bdf040ff1e317104fc7aa3857513b5b6b64b0)